lib/player.js

```diff
@@ -443,6 +443,56 @@ goog.requireType('shaka.media.PresentationTimeline');
  */
 
 
+/**
+ * @event shaka.Player.Started
+ * @description Fires when the content starts playing.
+ *     Only for VoD.
+ * @property {string} type
+ *   'started'
+ * @exportDoc
+ */
+
+
+/**
+ * @event shaka.Player.FirstQuartile
+ * @description Fires when the playhead crosses first quartile.
```

Comment:
Why does this JSDoc say "playhead", when the other middle-of-the-content events say "content playhead"?

Reply:
Fixed!